index.html

```diff
@@ -250,6 +250,31 @@ <h3 class="text-3">LET’S CONNECT</h3>
 
 
     </main>
+    <footer class="footer">
+        <div class="foot-image">
```

Comment:
### Your Border Radius
![upper-part (copy)](https://user-images.githubusercontent.com/93039359/187659188-6602a9c5-b1db-47c8-9419-1169aefe5a4f.png)

### Your Figma Border Radius
![border-radius-figma (copy)](https://user-images.githubusercontent.com/93039359/187659111-ccbd0571-021b-460d-b2ac-187c81d5e89d.png)

- [ ] Your Border radius is more than the Figma design border radius :sob: Please make it similar to the Figma template :+1: